app/Enums/WebhookAction.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Enums;
+
+enum WebhookAction: int
+{
+    case NEW = 1;
```

Comment:
Please use CREATED so it follows the same convention as the other two options :P